docs/rules/jsx-no-literals.md

```diff
@@ -26,16 +26,17 @@ var Hello = <div>
 
 ## Rule Options
 
-There are two options:
+There are three options:
```

Comment:
Probably a good idea to remove the count, to avoid having to update this in the future (also you're changing it to "three" but i think there's 4 now?):
```suggestion
The supported options are:
```

Reply:
Good catch, the wording has been changed.